tests/MyGame/Example/Any.py

```diff
@@ -8,3 +8,18 @@ class Any(object):
     TestSimpleTableWithEnum = 2
     MyGame_Example2_Monster = 3
 
+
+def AnyCreator(unionType, table):
+    from flatbuffers.table import Table
+    if not isinstance(table, Table):
+        return None
+    if unionType == Any().Monster:
+        import MyGame.Example.Monster
```

Comment:
Do you see a way to move these imports out of function bodies? Calling `import` has performance overhead we'd like to avoid: https://nyu-cds.github.io/python-performance-tips/09-import/

Reply:
I basically followed the convention of the existing API: https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example/Monster.py#L98. I guess the reason is that some flatbuffers may have lots of tables and structs. Thus, instead of importing all of them in advance, it only imports when a module is needed. 

We can definitely change it. But since it involves changes for all of the entire APIs, I wonder if we can do that in a separate PR.

Reply:
SGTM

Reply:
Thank you Robert!